include/git2/sys/textconv.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+#ifndef INCLUDE_sys_git_textconv_h__
+#define INCLUDE_sys_git_textconv_h__
+
+typedef struct git_textconv git_textconv;
+
+/**
+ * @file git2/sys/textconv.h
+ * @brief Git textconv backend and plugin routines
+ * @defgroup git_backend Git custom backend APIs
+ * @ingroup Git
+ * @{
+ */
+GIT_BEGIN_DECL
+
+/**
+ * Look up a textconv by name
+ *
+ * @param name The name of the textconv
+ * @return Pointer to the textconv object or NULL if not found
+ */
+GIT_EXTERN(git_textconv *) git_textconv_lookup(const char *name);
+
+#define GIT_TEXTCONV_HTTP  "http"
```

Comment:
Maybe I missed it, but I don't think there's actually a default textconv in there ?